sdk/communication/Azure.Communication.MediaComposition/tests/samples/Sample1_MediaCompositionClient.cs

```diff
@@ -70,7 +69,7 @@ public async Task UpdateLayoutMediaCompositionAsync()
             var mediaCompositionClient = CreateClient();
             await CreateMediaCompositionHelper(mediaCompositionClient);
             #region Snippet:UpdateLayout
-            var layout = new AutoGridLayout(new List<string>() { "teamsMeeting" })
+            var layout = new AutoGridLayout(new List<string>() { "acsGroupCall" })
```

Comment:
README snippets are derived from these #region Snippet:<name> tags. We should rerun the script to update the READMEs since this code has been modified. Please follow the instructions here: https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets

Reply:
Correct, updated the ReadMe.md by running the update scripts. Pushed the new changes.